### PR TITLE
Add missing whitespace.

### PR DESCRIPTION
**Pull request by stepshal**

### CODE REVIEW

File: setuptools/sandbox.py [hunk 1]
```diff
@@ -305,54 +305,54 @@ def wrap(self,path,*args,**kw):
...
-            self._remap_input(operation+'-from',src,*args,**kw),
-            self._remap_input(operation+'-to',dst,*args,**kw)
+            self._remap_input(operation+'-from', src, *args, **kw),
+            self._remap_input(operation+'-to', dst, *args, **kw)
         )
 
 
 if hasattr(os, 'devnull'):
-    _EXCEPTIONS = [os.devnull,]
+    _EXCEPTIONS = [os.devnull, ]
```
Comment (jaraco) [inline setuptools/sandbox.py]: I think I prefer the previous syntax. Is a trailing whitespace the recommended style? Can you provide a reference?

**Comment by jaraco** [comment]
Other than that one concern, this change looks good.

**Comment by stepshal** [comment]
@jarace Done.

**Comment by stepshal** [comment]
@jaraco Done.

**Comment by stepshal** [comment]
@jaraco Conflicts with base branch resolved.

**Pull request merged**